src/main/java/wolox/training/exceptions/handler/ErrorHandler.java

```diff
@@ -21,13 +27,32 @@ protected ErrorResponse BookNotFoundException(BookNotFoundException ex) {
                 ErrorsEnum.JSON_BOOK_NOT_FOUND_EXCEPTION.getMessage());
     }
 
+    /**
+     *
+     * @param ex: Exception BookIdMismathException
+     * @return ErrorResponse with the exception code and the message
+     */
     @ExceptionHandler(BookIdMismatchException.class)
-    @ResponseStatus(value = HttpStatus.NOT_FOUND)
+    @ResponseStatus(value = HttpStatus.BAD_REQUEST)
     @ResponseBody
     protected ErrorResponse BookIdMismatchException(BookIdMismatchException ex) {
 
         return new ErrorResponse(ErrorsEnum.JSON_BOOK_ID_MISMATCH_EXCEPTION.getCode(),
                 ErrorsEnum.JSON_BOOK_ID_MISMATCH_EXCEPTION.getMessage());
     }
 
+    /**
+     *
+     * @param ex: Exception BookAlreadyOwnedException
+     * @return ErrorResponse with the exception code and the message
+     */
+    @ExceptionHandler(BookAlreadyOwnedException.class)
+    @ResponseStatus(value = HttpStatus.BAD_REQUEST)
+    @ResponseBody
+    protected ErrorResponse BookAlreadyOwnedException(BookAlreadyOwnedException ex) {
```

Comment:
@MatiasVillalba96 The same comment about the method name must start in lowercase. 
😄